spec/system/users_spec.rb

```diff
@@ -1,124 +1,128 @@
-					click_button 'SignUp'
-					expect(page).to have_content "Email can't be blank"
-				end
+          visit sign_up_path
+          fill_in 'Email', with: user.email
+          fill_in 'Password', with: user.password
+          fill_in 'Password confirmation', with: user.password
+          click_button 'SignUp'
+          expect(page).to have_content "Email can't be blank"
+          expect(page).to have_current_path users_path, ignore_query: true
+        end
       end
 
       context '登録済のメールアドレスを使用した場合' do
-				let(:user) { create(:user) }
-				let(:another_user) { builde(:user) }
+        let(:user) { create(:user) }
+        let(:another_user) { builde(:user) }
+
         it 'ユーザーの新規作成が失敗する' do
-					visit sign_up_path
-					fill_in 'Email', with: user.email
-					fill_in 'Password', with: user.password
-					fill_in 'Password confirmation', with: user.password
-					click_button 'SignUp'
-					expect(page).to have_content "Email has already been taken"
-				end
+          visit sign_up_path
+          fill_in 'Email', with: user.email
+          fill_in 'Password', with: user.password
+          fill_in 'Password confirmation', with: user.password
+          click_button 'SignUp'
+          expect(page).to have_content 'Email has already been taken'
+          expect(page).to have_current_path users_path, ignore_query: true
+        end
       end
     end
 
     describe 'マイページ' do
       context 'ログインしていない状態' do
-				let(:user) { create(:user) }
-        	it 'マイページへのアクセスが失敗する' do
-					  visit user_path(user)
-					  expect(page).to have_current_path login_path
-				end
-      end
-		end
+        let(:user) { create(:user) }
 
+        it 'マイページへのアクセスが失敗する' do
+          visit user_path(user)
+          expect(page).to have_content 'Login required'
+          expect(page).to have_current_path login_path, ignore_query: true
+        end
+      end
+    end
   end
 
   describe 'ログイン後' do
     describe 'ユーザー編集' do
       context 'フォームの入力値が正常' do
-				let(:user) { create (:user) }
-				before { login(user) }
+        let(:user) { create(:user) }
+
+        before { login(user) }
+
         it 'ユーザーの編集が成功する' do
-					visit edit_user_path(user)
-					fill_in "Email", with: user.email
-					fill_in "Password", with: "pass"
-					fill_in "Password confirmation", with: "pass"
-					click_button "Update"
-					expect(page).to have_content "User was successfully updated."
-				end
+          visit edit_user_path(user)
+          fill_in 'Email', with: user.email
+          fill_in 'Password', with: 'pass'
+          fill_in 'Password confirmation', with: 'pass'
+          click_button 'Update'
+          expect(page).to have_content 'User was successfully updated.'
+          expect(page).to have_current_path user_path(user), ignore_query: true
+        end
       end
 
       context 'メールアドレスが未入力' do
-				let(:user) { create (:user) }
-				before { login(user) }
+        let(:user) { create(:user) }
+
+        before { login(user) }
+
```

Comment:
好みもありますが、1行開けなくてもいいかなと思います！
```suggestion
        let(:user) { create(:user) }
        before { login(user) }

```

Reply:
e87943b にてletとbeforeの間に1行開けていた書き方を修正しました。